### DISCUSSION TITLE
Stable Releases

### DISCUSSION THREAD

**Discussion by lukasdotcom** [kagemomiji/airsonic-advanced#312] [General]
The last stable release was in April, and you have been doing a lot of work on this project. I was wondering if you ever planning to release another stable release, or if I should just switch to the edge releases.

**Comment by fault1er1** [comment]
I'm (and lots of my friends are) using the edge release on a daily basis and it's as stable as the stable release as far as I can see. I have the same old problems like 100% CPU once every few weeks or months.

**Comment by kagemomiji** [comment]
@Lukasdotcom
Currently, I am undertaking a major refactoring to switch to JPA Repository, and this work is now approaching its final stages. Once the work is completed and a few bugs are fixed, I plan to execute the next Stable Release. I am aiming to do this within the year.

**Reply by dsiminiuk** [reply]
Very nice. I wrote a script that checks for Stable Releases weekly and it has been a while as @Lukasdotcom also noticed. I was going to ask the same, so thank you for that. @kagemomiji thank you for your work on this app, it's really great.

**Comment by masterwishx** [comment]
im using lasted in docker but when stable released its not same as lasted , should be the lasted also point to stable if it released ?

**Reply by kagemomiji** [reply]
@masterwishx 
It's because I've inherited the release mechanism from the original Airsonic-Advanced. I will make changes so that from the next stable release, 'latest' will point to 'stable'.
